tic-tac-toe.js

```diff
@@ -1,7 +1,158 @@
+  this.four = this.board[1][0];
+  this.five = this.board[1][1];
+  this.six = this.board[1][2];
+  this.seven = this.board[2][0];
+  this.eight = this.board[2][1];
+  this.nine = this.board[2][2];
 }
 
-TicTacToe.prototype = {
-  
-}
+TicTacToe.prototype.positionAsAttribute = function(position) {
+  switch (position) {
+    case '1':
+      return this.one;
+    case '2':
+      return this.two;
+    case '3':
+      return this.three;
+    case '4':
+      return this.four;
+    case '5':
+      return this.five;
+    case '6':
+      return this.six;
+    case '7':
+      return this.seven;
+    case '8':
+      return this.eight;
+    case '9':
+      return this.nine;
+  }
+};
+
+TicTacToe.prototype.switchTurn = function () {
+  if (this.turn === 'playerOne') {
+    this.turn = 'playerTwo';
+  } else if (this.turn === 'playerTwo') {
+    this.turn = 'playerOne';
+  }
+};
+
+TicTacToe.prototype.selectTilePlayerOne = function (position) {
+  switch (position) {
+    case '1':
+      this.one = 'x';
+      break;
+    case '2':
+      this.two = 'x';
+      break;
+    case '3':
+      this.three = 'x';
+      break;
+    case '4':
+      this.four = 'x';
+      break;
+    case '5':
+      this.five = 'x';
+      break;
+    case '6':
+      this.six = 'x';
+      break;
+    case '7':
+      this.seven = 'x';
+      break;
+    case '8':
+      this.eight = 'x';
+      break;
+    case '9':
+      this.nine = 'x';
+      break;
+  }
+};
+
+TicTacToe.prototype.selectTilePlayerTwo = function (position) {
+  switch (position) {
+    case '1':
+      this.one = 'o';
+      break;
+    case '2':
+      this.two = 'o';
+      break;
+    case '3':
+      this.three = 'o';
+      break;
+    case '4':
+      this.four = 'o';
+      break;
+    case '5':
+      this.five = 'o';
+      break;
+    case '6':
+      this.six = 'o';
+      break;
+    case '7':
+      this.seven = 'o';
+      break;
+    case '8':
+      this.eight = 'o';
+      break;
+    case '9':
+      this.nine = 'o';
+      break;
+  }
+};
+
+TicTacToe.prototype.gameWon = function() {
+  var t = this;
+  var horizontalWin = function () {
+    if (t.one === t.two && t.two === t.three && t.one !== 0) {
+      return true;
+    } else if (t.three === t.four && t.four === t.five && t.three !== 0) {
+      return true;
+    } else if (t.seven === t.eight && t.eight === t.nine && t.seven !== 0) {
+      return true;
+    }
+  };
+  var verticalWin = function () {
+    if (t.one === t.four && t.four === t.seven && t.one !== 0)  {
+      return true;
+    } else if (t.two === t.five && t.five === t.eight && t.two !== 0) {
+      return true;
+    } else if (t.three === t.six && t.six === t.nine && t.three !== 0) {
+      return true;
+    }
+  };
+  var diagonalWin = function () {
+    if (t.one === t.five && t.five === t.nine && t.one !== 0) {
+      return true;
+    } else if (t.three === t.five && t.five === t.seven && t.three !== 0) {
+      return true;
+    }
```

Comment:
this works given your current implementation, but it is a lot of `&&` checks and if else statements, if the game changed to a 9x9 structure, for example, it would be very difficult to implement in this way.